lib/date_item_widget.dart

```diff
@@ -64,7 +71,9 @@ class DateItemWidget extends StatelessWidget {
               List<Widget>.generate(this.dateItemComponentList.length, (index) {
             switch (this.dateItemComponentList[index]) {
               case DateItem.WeekDay:
-                return Text(DateFormat.E().format(this.dateTime),
+                return Text(
+                    DateFormat.E(Locale(this.locale).toString())
+                        .format(this.dateTime),
```

Comment:
Which flutter version are you using? I am using 2.2.3 and the it keeps warning me `The argument type 'String?' can't be assigned to the parameter type 'String'.`